frontend/src/i18n/locales/en-us.json

```
@@ -62,6 +62,7 @@
     "question-circle-warning": "Don't use this feature unless you know what you're doing.",
     "learn-more": "Learn more",
     "select-token": "Select token to send",
+    "slippage-exceeded": "Actual send amount would materially differ from estimate. Please retry.",
```

Nit, feels a little more clear, lmk what you think / feel free to resolve this comment yourself. Also @garyghayrat can you give us the translation for whatever @davidlaprade goes with? 
```suggestion
    "slippage-exceeded": "Actual send amount would significantly differ from estimate due to changing gas prices. Please try again.",
```

---

Eh reading this again now I get why you used "materially". My motivation for changing it was that it seemed like a less common word for users who aren't native english speakers, but I don't know if that's actually true

---

I think "material" is pretty common in finance-speak, but I have no idea how recognizable it is to non-native speakers. Curious what Gary thinks. For now I'll leave it but take the other suggestions, thanks!

Also, yeah, @garyghayrat if you could give me the translation for Chinese, that'd be great!

---

I like matt's suggestion overall. Like the simpler 'significantly' and the explanation of `due to changing gas prices.` Just added the translation as well.

---

Looks like we still have "materially"? I'm ok with that if that's the decision, but from the above thought we were going with "significantly"